packages/core/src/api.js

```diff
@@ -118,8 +118,9 @@ export function createPercyServer(percy, port) {
     }))
     .route('post', '/percy/automateScreenshot', async (req, res) => {
       req = percyAutomateRequestHandler(req, percy.build);
+      percy.upload(await new WebdriverUtils(req.body).automateScreenshot());
```

Comment:
check #1369 

I've tried this, though this doesn't make it async, since we need to have a way to make that `await` removed which will require some more code changes. though this version is much simplified then what it is in master.

Reply:
Yes this is not full await however this unblocks PoA SDK. CLI can keep on waiting but if we add await which is master as of now that blocks the PoA. So if we have this thing then we do see speed improvements.

Reply:
you can close this PR, as I've updated this in the linked PR. 

Also can you please review other bit in that PR?